src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Migration Strategy: Lift and Shift"
+summary: Learn about the 'Lift and Shift' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+There are multiple strategies for [migrating off legacy technology](migration-overview.html#molt) to CockroachDB.
+
+This page discusses the 'Lift and Shift' strategy for migrating your data, which is a commonly used approach. This approach, which is also known as 'Big Bang' (and by other names), refers to the process where your data is moved in its entirety from a source system to a target system within a defined period of time. This can involve some application downtime or service degradation.
+
+Lift and Shift is a commonly used approach. However, it may not be the right approach if a strong application service continuity during the migration is required. It may be a viable method if application downtime is permitted.
```

Comment:
You already say commonly used in the preceding paragraph.

I'd suggest deleting and starting this paragraph with "Lift and shift may not be the right approach..."

Reply:
+1

Reply:
Done - thanks